### DISCUSSION TITLE
Failed to get size for image: https://api.web3modal.com/public/getAssetImage/41d04d42-da3b-4453-8506-668cc0727900

### DISCUSSION THREAD

**Discussion by ManiMohan-Radhakrishnan** [WalletConnect/walletconnect-monorepo#4261] [Developer Support] [answered]
How to fix ???

Failed to get size for image: https://api.web3modal.com/public/getAssetImage/41d04d42-da3b-4453-8506-668cc0727900
<img width="1440" alt="Screenshot 2024-02-23 at 7 23 03 PM" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/56622784/d6d27606-5f0b-4a30-94f8-f53396cb33cb">

**Comment by glitch-txs** [comment]
hi, could you share a minimal reproducible example?

**Comment by ml4st** [comment]
Same problem here, works like a charm on iOS simulator, but for some reason wallets/networks images fail to load on Android (Expo Go on my android device, as well as Android Studio emulator).

You can find my project here :
https://github.com/matthieup416/expo-wallet-connect

![Screenshot_20240312_133347](https://github.com/WalletConnect/walletconnect-monorepo/assets/83717963/26a398cc-5a00-471d-835f-919abbeae9f5)

**Comment by ignaciosantise** [comment, marked as answer]
Hi @matthieup416 @ManiMohan-Radhakrishnan 👋  The fix is available in the latest version of the SDK! Can you try updating to `v1.2.1`?

Also @matthieup416 thanks for the example project 💪

**Reply by ml4st** [reply]
Thanks @ignaciosantise updating  to v1.2.1 fixed the issue
